op-service/txmgr/txmgr.go

```diff
@@ -905,3 +905,87 @@ func finishBlobTx(message *types.BlobTx, chainID, tip, fee, blobFee, value *big.
 	}
 	return nil
 }
+
+type TestTxManager struct {
```

Comment:
would be nice to move this type and its methods into a new file `test_txmgr.go`.

Reply:
done.. should we also move TestBatchSubmitter into its own file?

Reply:
good idea! sorry, missed that

Reply:
moved TestBatchSubmitter into its own file too